packages/twenty-front/src/modules/settings/data-model/utils/getFieldDefaultPreviewValue.ts

```diff
@@ -31,6 +31,14 @@ export const getFieldDefaultPreviewValue = ({
     return defaultSelectOption.value;
   }
 
+  // Multi-select field
```

Comment:
this comment is not useful!

Reply:
Ok, I will remove the other onces then:
![image](https://github.com/twentyhq/twenty/assets/29927851/5d5ac7d1-fb55-4d69-8bba-c636f4a1b9d9)